modules/ROOT/pages/dataweave-flow-control-precedence.adoc

```diff
@@ -106,7 +106,7 @@ in order from the first to the last function specified in the chain.
 
 For example, the following script acts on the input array of objects defined
 by the variable `flights`. The script first calls `filter` to return an array
-that filters out all `price` values that are less than 500. Then it calls
+that filters out all `price` values that are greater than or equal to 500. Then it calls
```

Comment:
This is a double negation is harder to understand as the code says less than 500. So I would say `The script first calls `filter` to return an array that accepts all `price` values that are less than 500.`

Reply:
done